sources/packages/web/src/components/common/notes/Notes.vue

```diff
@@ -1,29 +1,28 @@
 <template>
-  <v-card>
-    <v-container :fluid="true">
-      <v-row class="m-2">
-        <v-col class="category-header-large color-blue">{{ title }}</v-col>
-        <v-col>
-          <check-permission-role :role="allowedRole">
-            <template #="{ notAllowed }">
-              <v-btn
-                @click="addNewNote"
-                class="float-right"
-                color="primary"
-                prepend-icon="fa:far fa-edit"
-                :disabled="notAllowed"
-              >
-                Create new note
-              </v-btn>
-            </template>
-          </check-permission-role>
-        </v-col>
-      </v-row>
-      <v-row class="m-2" v-if="!notes || notes.length === 0"
-        ><v-col
-          >No notes found. Please click on create new note to add one.</v-col
-        ></v-row
-      >
+  <v-container :fluid="true">
+    <v-row class="m-2">
+      <v-col class="category-header-large color-blue">{{ title }}</v-col>
```

Comment:
This file has formatting warnings. 

![image](https://user-images.githubusercontent.com/54600590/226406529-fb196e35-cf85-4548-b39d-e738fb021486.png)
![image](https://user-images.githubusercontent.com/54600590/226406970-ed047117-9a68-42c4-81bd-7e666b802bf9.png)